README.md

```diff
@@ -39,6 +43,13 @@ you can of course build your own. (The `example/eventsource-polyfill.js` is buil
 
 ## Extensions to the W3C API
 
+### Extensions to event behaviour
+
+Emits new `retrying` event when the library is attempting to reconnect to the Harness.
```

Comment:
This library is not Harness-specific, so we can drop what it is trying to reconnect to.
```suggestion
Emits new `retrying` event when the library is attempting to reconnect.
```